djautotask/models.py

```diff
@@ -520,3 +523,18 @@ def get_entered_time(self):
             entered_time = date_worked - local_offset
 
         return entered_time
+
+
+class Role(models.Model):
+    name = models.TextField(blank=True, null=True, max_length=200)
```

Comment:
Should this be a CharField instead since its max length is only 200? Same goes for description.